docker/grpc.Dockerfile

```diff
@@ -35,7 +35,9 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 	libjansson-dev \
         libcjose-dev \
         libgtest-dev \
-        libgmock-dev
+        libgmock-dev \
+        python3-pip
+RUN pip3 install behave PyHamcrest pathlib2
```

Comment:
Seria possível que essa instalação fosse feita a partir de _integration_test/requirements.txt_ ?
Para chamar algo tipo `python3 -m pip install -r requirements.txt` e as dependências de teste ficarem centralizadas num só lugar

Reply:
Essa é nossa imagem base, o ideal é que ela tenha todas nossas dependências para diminuir o tempo de pipeline

Reply:
So, I think what @arianalima  means is when building the image, we could put in some file all requirements and I agree with her.
We could do the same for apt depencies.